assets/js/components/ClusterDetails/ChecksResults.jsx

```diff
@@ -253,6 +237,14 @@ export const ChecksResults = () => {
           }
         />
       </div>
+      {cluster.provider == 'unknown' && (
```

Comment:
You can do

```js
const UNKNOWN = 'unknown';
```

on top of the file and use it here as

```js
cluster.provider == UNKNOWN &&
```

it would be a little more idiomatic.

Reply:
Done: https://github.com/trento-project/web/pull/840/commits/a9156c6d81d1a1365cfe173f8bb0e6cc8c133912